cluster-stamp.bicep

```diff
@@ -43,7 +43,7 @@ param clusterAuthorizedIPRanges array = []
   'southeastasia'
 ])
 param location string = 'eastus2'
-param kubernetesVersion string = '1.22.4'
+param kubernetesVersion string = '1.23.3'
```

Comment:
Sorry, one more thing .. and this is a blocker.  Can this please be 1.23.5 instead.  1.23.3 is going to be deprecated with the next patch update, and I'd like to not have to chase that.  We could consider, for the first time, going with `1.23` and having a comment in here that says, # For intentionality you'll likely instead wish to pin this to a specific version, such as 1.23.5.

Reply:
I think we had discussed that once this would be available is what we do wanted to implement. However, I have no personal preference for one or the another.

Im seeing that handling specific versions will require more maintenance effort but at the same time help us to have more control over what the clusters version are being deployed which is better for troubleshooting.

I would let you make the final call in here.

Reply:
in the meantime we are upgrading to `1.23.5` @ckittel 

done | addressed from 73ac8ad3499e0da1c8c5099c92eb20939dab0090

Reply:
I'm good with 1.23.5 for now.  Thanks!